### PR TITLE
added steam command

### PR DESCRIPTION
**Pull request by almeidx**
A command that searches for games on the steam store

### CODE REVIEW

File: src/apis/SteamStore.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    })
+  }
+
+  search (query) {
+    return axios({
+      url: `${API_URL}/storesearch`,
+      params: {
+        term: query,
+        l: 'english',
+        cc: 'US'
```
Comment (metehus) [inline src/apis/SteamStore.js:19]: can't this be relative?

Comment (almeidx) [reply src/apis/SteamStore.js:19]: it only accepts 2 char language codes, but I can try

Comment (almeidx) [reply src/apis/SteamStore.js:19]: I could use the `l` param on the app info tho, the only issue is that it doesn't support values such as `pt-BR` or whatever, it only supports the full language name, like `english` or `portuguese`

Comment (metehus) [reply src/apis/SteamStore.js:19]: https://www.npmjs.com/package/iso-639 (this one is a bit heavy, there's more out there)
get the language code (`pt`), find on the table and get the name (`Portuguese`), lowercase it (`portuguese`), and use it on the `l`, and you can try to get the country (`BR`) to use on the `cc`. Some languages on the bot(like the furry one), just use english as default, and also remember to test the api to see if it works with unknown value (e.g. using random string on the `l` and `cc` to see if return error or just defaults to english) to handle future errors

Comment (almeidx) [reply src/apis/SteamStore.js:19]: https://github.com/SwitchbladeBot/switchblade/pull/1242/commits/8f9a13de63af7e1e1f790586427536a5409ea338

**Review by Doges — DISMISSED** [review]
LGTM

**Review by fernando457829 — APPROVED** [review]
LGTM

File: src/commands/misc/steam.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      aliases: ['steamstore'],
+      requirements: { apis: ['steamstore'] },
+      embedColor: Constants.STEAM_COLOR,
+      embedLogoURL: 'https://i.imgur.com/PA1NIoK.png'
+    }, client)
+  }
+
+  async search (_, query) {
+    const res = await this.client.apis.steamstore.search(query)
+    return res.data.items
```
Comment (metehus) [inline src/commands/misc/steam.js:18]:
```suggetion
    return this.client.apis.steamstore.search(query)
      .then(({ data }) => data.items)
```

**Comment by Tsugami** [comment]
LGTM

**Review by Tsugami — APPROVED** [review]

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**